oracle.py

```diff
@@ -32,11 +33,12 @@ def update_legality(self, legal_cards):
 
     def update_database(self, new_version):
         self.database.execute('DELETE FROM version')
-        self.database.execute('DELETE FROM card')
```

Comment:
We'd need to drop all printing, subtype, etc. info for that to work, wouldn't we? Or is sqlite sophisticated about cascading foreign keys? Seems unlikely?